### PR TITLE
Georgian number and currency transformers

### PR DESCRIPTION
**Pull request by goodot**
I used some logic from this  [repo](https://github.com/d0ragon/number-to-words-ka).

### CODE REVIEW

File: src/Legacy/Numbers/Words/Locale/Ka.php [hunk 1]
```diff
@@ -0,0 +1,185 @@
+<?php
+/**
```
Comment (kwn) [inline src/Legacy/Numbers/Words/Locale/Ka.php]: Could you please remove the docblock from there?

File: src/NumberToWords.php [hunk 2]
```diff
@@ -70,7 +72,8 @@ class NumberToWords
         'sv' => SwedishNumberTransformer::class,
         'tk' => TurkmenNumberTransformer::class,
         'tr' => TurkishNumberTransformer::class,
-        'ua' => UkrainianNumberTransformer::class
+        'ua' => UkrainianNumberTransformer::class,
+        'ka' => GeorgianNumberTransformer::class
```
Comment (kwn) [inline src/NumberToWords.php]: Keys in the array are sorted alphabetically. Could you please move the `'ka' => GeorgianNumberTransformer::class` above?

File: src/NumberToWords.php [hunk 3]
```diff
@@ -87,7 +90,8 @@ class NumberToWords
         'sk' => SlovakCurrencyTransformer::class,
         'tk' => TurkmenCurrencyTransformer::class,
         'tr' => TurkishCurrencyTransformer::class,
-        'ua' => UkrainianCurrencyTransformer::class
+        'ua' => UkrainianCurrencyTransformer::class,
+        'ka' => GeorgianCurrencyTransformer::class
```
Comment (kwn) [inline src/NumberToWords.php]: Keys in the array are sorted alphabetically. Could you please move the `'ka' => GeorgianCurrencyTransformer::class` above?

File: src/NumberTransformer/GeorgianNumberTransformer.php [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace NumberToWords\NumberTransformer;
+
+use NumberToWords\Legacy\Numbers\Words;
+
+class GeorgianNumberTransformer implements NumberTransformer
+{
+
```
Comment (kwn) [inline src/NumberTransformer/GeorgianNumberTransformer.php]:
```suggestion
```

File: tests/CurrencyTransformer/GeorgianCurrencyTransformerTest.php [hunk 5]
```diff
@@ -0,0 +1,33 @@
+<?php
+/**
```
Comment (kwn) [inline tests/CurrencyTransformer/GeorgianCurrencyTransformerTest.php]: Could you please remove the docblock? It doesn't add any value.

File: tests/CurrencyTransformer/GeorgianCurrencyTransformerTest.php [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+ * Created by PhpStorm.
+ * User: godot
+ * Date: 7/20/19
+ * Time: 12:11 AM
+ */
+
+namespace NumberToWords\CurrencyTransformer;
+
```
Comment (kwn) [inline tests/CurrencyTransformer/GeorgianCurrencyTransformerTest.php]:
```suggestion
```

File: tests/CurrencyTransformer/GeorgianCurrencyTransformerTest.php [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+class GeorgianCurrencyTransformerTest extends CurrencyTransformerTest
+{
+    public function setUp()
+    {
+        $this->currencyTransformer = new GeorgianCurrencyTransformer();
+    }
+
```
Comment (kwn) [inline tests/CurrencyTransformer/GeorgianCurrencyTransformerTest.php]:
```suggestion
```

File: tests/CurrencyTransformer/GeorgianCurrencyTransformerTest.php [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+    public function providerItConvertsMoneyAmountToWords()
+    {
+        return [
+            [1025, 'USD', 'ათი დოლარი და ოცდახუთი ცენტი'],
```
Comment (kwn) [inline tests/CurrencyTransformer/GeorgianCurrencyTransformerTest.php]: Could you please order test cases ascending?

File: tests/CurrencyTransformer/GeorgianCurrencyTransformerTest.php [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+            [105, 'GEL', 'ერთი ლარი და ხუთი თეთრი'],
+            [2001, 'EUR', 'ოცი ევრო და ერთი სენტიმი'],
+            [1, 'GEL', 'ერთი თეთრი'],
+            [125, 'TRY', 'ერთი ლირა და ოცდახუთი ყურუში'],
+            [1050, 'AMD', 'ათი დრამი და ორმოცდაათი ლუმა'],
+            [180, 'PLN', 'ერთი ზლოტი და ოთხმოცი გროში'],
+            [121084, 'GBP', 'ათას ორას ათი ფუნტი და ოთხმოცდაოთხი პენი']
+        ];
+    }
+}
```
Comment (kwn) [inline tests/CurrencyTransformer/GeorgianCurrencyTransformerTest.php]:
```suggestion
}

```

File: tests/NumberTransformer/GeorgianNumberTransformerTest.php [hunk 10]
```diff
@@ -0,0 +1,132 @@
+<?php
+/**
```
Comment (kwn) [inline tests/NumberTransformer/GeorgianNumberTransformerTest.php]: Same as above, thanks.

File: tests/NumberTransformer/GeorgianNumberTransformerTest.php [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+ * Created by PhpStorm.
+ * User: goderdzi.jobadze
+ * Date: 7/19/2019
+ * Time: 5:46 PM
+ */
+
+namespace NumberToWords\NumberTransformer;
+
```
Comment (kwn) [inline tests/NumberTransformer/GeorgianNumberTransformerTest.php]:
```suggestion
```

**Comment by kwn** [comment]
Hi @goodot 
Thank you for the great contribution! Could you also please update the README.md file and add Georgian language there?

PS. Georgian alphabet looks amazing!

**Comment by goodot** [comment]
Done! Thanks for praising Georgian alphabet, happy to hear :)) მადლობა

**Comment by kwn** [comment]
Once again thank you for the contribution!!

**Comment by goodot** [comment]
Welcome

**Pull request merged**